resources/js/components/UserEditForm.vue

```diff
@@ -326,7 +335,18 @@ export default {
         );
       });
   },
+  beforeDestroy() {
+    window.removeEventListener('beforeunload', this.onLeaveAttempt);
+  },
   methods: {
+    onFormChange() {
+      this.formChanged = true;
+    },
+    onLeaveAttempt(event) {
+      if (this.formChanged) {
+        event.returnValue = "Are you sure you want to leave? Any unsaved changes will be lost.";
```

Comment:
I thought the prompt choices, at least in chrome, are ok/cancel, which doesn't align with the question asked here